### DISCUSSION TITLE
AttributeError: module 'tvm.script.tir' has no attribute 'Buffer'问题

### DISCUSSION THREAD

**Discussion by tiantang5156** [mlc-ai/mlc-zh#57] [Q&A]
对照练习中调用tvm.script.tir.Buffer，发现报了如标题所见的错误，打开tir也看到有Buffer类的实现，请问有老师或同学能解答一下吗，不胜感激。

**Comment by DzAvril** [comment]
建议你把你运行的完整的Python脚本放上来以方便查找问题

**Comment by lhumanfy** [comment]
遇到了一样的问题，请问你解决了吗

**Reply by lhumanfy** [reply]
我已经知道为什么了，使用python3 -m  pip install mlc-ai-nightly -f https://mlc.ai/wheels  安装的环境是没有问题，在源码里可以看到这样一句话。而使用源码包安装没有这个类型所以报错了
<img width="394" alt="image" src="https://user-images.githubusercontent.com/57166664/182522476-24ac41f2-5da1-45dc-9df2-acc0d0e8fc9a.png">
